styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/ec-tabs-course-browser.twig

```diff
@@ -0,0 +1,21 @@
+<div class="tabs-course_browser">
+  {% include 'atoms-h1' with { "class": "tabs-course_browser_label", "content": "Browse Our Top Courses" } %}
+  <ul class="tabs-course_browser_list-tabs" role="menu">
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-1" role="menuitem">Clinical</a></li>
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-2" role="menuitem">Workflow &amp; Process</a></li>
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-3" role="menuitem">Patient Care</a></li>
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-4" role="menuitem">Leading Change</a></li>
+    <li class="tabs-course_browser_list_item"><a href="{{ url }}" data-tab="tab-5" role="menuitem">Personal Well-being</a></li>
+  </ul>
+  <ul class="tabs-course_browser_list-cards layout layout-four-up">
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+    <li class="tabs-course_browser_list-cards-item">{% include 'molecules-ec-card-course' %}</li>
+  </ul>
+  <div class="tabs-course_browser_ctas">
+    {% include 'atoms-button-link' with { 'url': "https://searchec.ama-assn.org", 'class': "tabs-course_browser_cta-primary", "content": "Start a Course" } %}
```

Comment:
Please add the `button-small` class for both of these. I don't think we're using the standard buttons anywhere at the moment.